standalone_app/src/sqlite3.ts

```diff
@@ -233,15 +273,26 @@ const paramInternalValueToExternalValue = (
 
 const parseDistributionJSON = (t: string): Distribution => {
   const parsed = JSON.parse(t)
-  if (parsed.name === "FloatDistribution") {
+  const floatDistributionList = [
+    "FloatDistribution",
+    "UniformDistribution",
+    "LogUniformDistribution",
+    "DiscreteUniformDistribution",
+  ]
+  const intDistributionList = [
+    "IntDistribution",
+    "IntUniformDistribution",
+    "IntLogUniformDistribution",
+  ]
+  if (floatDistributionList.includes(parsed.name)) {
```

Comment:
The attributes of UniformDistribution, LogUniformDistribution, and DiscreteUniformDistribution, are bit different with FloatDistribution, so that we need to write this logic like below.

```python
if (parsed.name === "FloatDistribution") {
  return {
      type: "FloatDistribution",
      low: parsed.attributes.low as number,
      high: parsed.attributes.high as number,
      step: parsed.attributes.step as number,
      log: parsed.attributes.log as boolean,
    }
} else if (parsed.name === "UniformDistribution") {
  return {
      type: "FloatDistribution",
      low: parsed.attributes.low as number,
      high: parsed.attributes.high as number,
      step: null,
      log: false,
    }
} else if (parsed.name === "LogUniformDistribution") {
  return {
      type: "FloatDistribution",
      low: parsed.attributes.low as number,
      high: parsed.attributes.high as number,
      step: null,
      log: true,
    }
} else if (parsed.name === "DiscreteUniformDistribution") {
  return {
      type: "FloatDistribution",
      low: parsed.attributes.low as number,
      high: parsed.attributes.high as number,
      step: parsed.attributes.q,
      log: false,
    }
} else if ...
```

Reply:
Understood. I'll rewrite the codes.